source/_integrations/number.mqtt.markdown

```diff
@@ -0,0 +1,126 @@
+---
+title: "MQTT Number"
+description: "Instructions on how to interact with a device exposing a Number through MQQT from within Home Assistant."
+ha_category:
+  - Number
+ha_release: 2020.02
+ha_iot_class: Configurable
+ha_domain: mqtt
+---
+
+The `mqtt` Number platform allows you to integrate devices that might expose configuration options through MQTT into Home Assistant as a Number. Every time a message under the `topic` in the configuration is received, the number entity will be updated in Home Assisant and vice-versa, keeping the device and Home Assistant in-sync.
+
+## Configuration
+
+To enable MQTT Number in your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Example configuration.yaml entry
+number:
+  - platform: mqtt
+    command_topic: my-device/threshold
+```
+
+{% configuration %}
+availability:
+  description: A list of MQTT topics subscribed to receive availability (online/offline) updates. Must not be used together with `availability_topic`.
+  required: false
+  type: list
```

Comment:
Given that you list keys, shouldn't this be a map rather than a list?

Reply:
I must confess this was copy & paste from existing mqtt documentation 😬

Reply:
It's a list of maps, the existing documentation is correct.